tests/Oryx.Integration.Tests/StorageAccountSanityTestBase.cs

```diff
@@ -331,10 +331,10 @@ private List<string> GetListOfVersionsToBuild(params string[] platformPath)
             using (var streamReader = new StreamReader(versionFile))
             {
                 string line = null;
-                while ((line = streamReader.ReadLine()) != null && !string.IsNullOrEmpty(line))
+                while ((line = streamReader.ReadLine()) != null)
                 {
-                    // ignore comments
-                    if (line.StartsWith("#"))
+                    // ignore comments or empty lines
+                    if (line.StartsWith("#") || string.IsNullOrEmpty(line))
```

Comment:
This is a good catch, thanks for making this change -- one thing I'm noticing we could do to expand this a little is for trim the `line` variable of any extra whitespace it is (specifically the case where the line _only_ has whitespace) to prevent it from skipping over the conditional we have here.

```cs
// Remove extraneous whitespace
line = line.Trim();

// Ignore comments or empty lines
if (line.StartsWith("#") || string.IsNullOrEmpty(line))
```

Reply:
Yes, here is the related work item: https://devdiv.visualstudio.com/DevDiv/_boards/board/t/Oryx/Stories/?workitem=1577218

Reply:
All are good suggestions. Parameterizing the pipeline does seem like it will solve the rollout case you are referring, Paul.

Reply:
Ok yeah that seems like it would fix the issue as long as we also make sure to have it override whatever is passed into the SDK storage tests, such as the prod url here https://github.com/microsoft/Oryx/blob/main/tests/Oryx.Integration.Tests/ProdStorageAccountSanityTest.cs#L20